packages/sources/kaiko/README.md

```diff
@@ -6,10 +6,11 @@ This document was generated automatically. Please see [README Generator](../../s
 
 ## Environment Variables
 
-| Required? |     Name     |      Description       |  Type  | Options |                      Default                       |
-| :-------: | :----------: | :--------------------: | :----: | :-----: | :------------------------------------------------: |
-|    ✅     |   API_KEY    |   API KEY for KAIKO    | string |         |                                                    |
-|           | API_ENDPOINT | API endpoint for KAIKO | string |         | `https://us.market-api.kaiko.io/v2/data/trades.v2` |
+| Required? |       Name        |         Description         |  Type  | Options |                      Default                       |
+| :-------: | :---------------: | :-------------------------: | :----: | :-----: | :------------------------------------------------: |
+|    ✅     |      API_KEY      |      API KEY for KAIKO      | string |         |                                                    |
+|           |   API_ENDPOINT    |   API endpoint for KAIKO    | string |         | `https://us.market-api.kaiko.io/v2/data/trades.v2` |
+|           | BASE_API_ENDPOINT | Base API endpoint for KAIKO | string |         |      `https://us.market-api.kaiko.io/v2/data`      |
```

Comment:
Yeah, good point. Remnant of an earlier commit.
We should probably make sure none of the NOPs are overriding the `API_ENDPOINT` env var first though, otherwise I'll probably have to undo the change in `transport/trades.ts`.

Related discussion from the Kaiko endpoint migration ticket: https://smartcontract-it.atlassian.net/browse/DF-19156

Reply:
> We should probably make sure none of the NOPs are overriding the API_ENDPOINT env var first though, otherwise I'll probably have to undo the change in transport/trades.ts.

Agree that ideally no NOPs are overriding the `API_ENDPOINT`, however if they are presumably the EA would error as the URL would be invalid? That might be acceptable as it will trigger the NOPs to update their config.

Reply:
Technically we could handle both URL formats if used a `URL` object (e.g. `new URL("https://us.market-api.kaiko.io/v2/data/trades.v2")`), as we could then extract the `host` and `pathname`.

However I'm not sure this is worth the complexity for a one-off deployment issue.

Reply:
closing the loop: agreed to set `API_ENDPOINT` as `https://us.market-api.kaiko.io/v2/data`, will follow up with PMs to determine what to do for NOPs